### DISCUSSION TITLE
ubuntu-drivers devices 명령어에 아무 반응도 일어나지 않습니다.

### DISCUSSION THREAD

**Discussion by hbm06047** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#50] [2024년도 교육 자료 관련 질문]
9번 영상대로 명령어를 입력하였지만 아무런 반응도 일어나지 않습니다. 관리자 권한으로 실행해도 똑같습니다. 어떻게 해야 할까요?

**Comment by Jinsun-Lee** [comment]
학생분께 어떠한 문제가 있는 건지 명확히 전달되지 않습니다. 어떠한 명령어를 입력했을 때, 아무런 동작도 하지 않았는지에 대해 사진을 캡처해 공유해 주시면 적절한 도움을 드리겠습니다.

**Reply by hbm06047** [reply]
![스크린샷 2024-07-14 21-47-29](https://github.com/user-attachments/assets/092c5ce2-c33e-4043-b1c8-15d20084cd82)
위 그림과 같이 nvidia-smi 명령어는 실패했다고 나타나고, ubuntu-drivers devices 명령어는 아예 작동을 하지 않고 끝나버립니다.
처음 ubuntu-drivers devices 명령어를 입력했을 때 강의 내용처럼 드라이버 목록이 나타나지 않아 강의처럼 535버전을 깔고 리부트했는데 nvidia-smi 명령어에 저렇게 동작하는 것 같습니다.

**Comment by Jinsun-Lee** [comment]
#### 1. 현재 노트북에 NVIDIA사의 내장 GPU가 없거나, GPU가 시스템에서 인식되지 않는 것으로 보입니다.
- 오류 메시지: ```nvidia-smi has failed because it couldn't communicate with the NVIDIA driver```
- 자율주행 통합 과정에서는 모든 학생에게 동일한 노트북이 제공될 예정이니 걱정하지 않으셔도 됩니다.

</br>

#### 2. 이 문제는 과제 수행과는 직접적인 연관이 없으므로 드라이버 설치는 생략하셔도 됩니다.
- 과제 수행과 관련된 노드에서는 CPU를 사용하도록 초기 세팅을 하였으며, 아래 코드에서 확인 가능합니다.
- [`src/camera_perception_pkg/camera_perception_pkg/yolov8_node.py`](https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/blob/main/src/camera_perception_pkg/camera_perception_pkg/yolov8_node.py)
